packages/__docs__/globals.js

```diff
@@ -82,7 +86,10 @@ const globals = {
   useEffect,
   useState,
   useRef,
-  ThemeColors
+  additionalPrimitives,
+  datavisualization,
```

Comment:
Consider camelCase for this as it is seen in other names.

Reply:
Nice catch, updated